tests/transfer-item.test.js

```diff
@@ -0,0 +1,130 @@
+const { connect, JSONCodec } = require("nats");
```

Comment:
Given (describe) When (describe) Then (it)

- Given a warehouse item is assigned to an inventory -> creer item, creer inventaire origine, assign inventory item
- When I transfer the inventory item to a new user -> creer nouvel inventaire, transferer
- Then the item has been transfered to the new user -> expect()
- Then the item is removed from the old inventory -> expect()